modules/proc-configuring-container-registry-authn.adoc

```diff
@@ -0,0 +1,78 @@
+// Module included in the following:
+//
+// @smccarthy-ie - Getting Started with Camel Kafka Connector
+//
+// Attributes used:
+//
+// {ProductName}
+// {OCPVersion}
+
+
+[id="configuring-container-registry-authn_{context}"]
+= Authenticating with Red Hat for container images
```

Comment:
Authenticating with the Red Hat Container Registry maybe? or Ecosystem Catalog

Reply:
Thanks Don. Yes, this is an interesting one. There was an initiative to clarify registry naming and the latest recommendation is very generic, e.g., "pulling images from Red Hat" (see https://docs.google.com/document/d/1kA633C0XwvWXpsZyuDxwrWLdE85HbR9SfVqq2EuNH-s/edit#). I've changed it back to authenticating with `registry.redhat.io`, which does exactly what it says on the tin :)

Reply:
TIL. thanks, Stephen!